src/main/java/uk/gov/hmcts/darts/task/runner/impl/ApplyRetentionAutomatedTask.java

```diff
@@ -32,8 +34,13 @@ public AutomatedTaskName getAutomatedTaskName() {
         return APPLY_RETENTION_TASK_NAME;
     }
 
+    @Override
+    public Duration getLockAtMostFor() {
+        return Duration.ofMinutes(90);
```

Comment:
As a general comment, would there be value in making these lock durations configurable via spring properties? Might be a useful thing to have easily tweakable without requiring a new code release?

Reply:
Updated :)